python_modules/dagster/dagster/_grpc/server.py

```diff
@@ -196,36 +200,40 @@ def __init__(
             # empty workspace
             return
 
-        loadable_targets = get_loadable_targets(
-            loadable_target_origin.python_file,
-            loadable_target_origin.module_name,
-            loadable_target_origin.package_name,
-            loadable_target_origin.working_directory,
-            loadable_target_origin.attribute,
-        )
-        for loadable_target in loadable_targets:
-            pointer = _get_code_pointer(loadable_target_origin, loadable_target)
-            recon_repo = ReconstructableRepository(
-                pointer,
-                container_image,
-                sys.executable,
-                entry_point=entry_point,
+        with user_code_error_boundary(
+            DagsterUserCodeLoadError,
+            lambda: "Error occurred during the loading of repository",
+        ):
```

Comment:
if you move this scoping down to tighter around just [2] you could probably have this error message refer to the code pointer of whats being loaded for better error messages via `CodePointer.describe`